package.json

```diff
@@ -75,7 +75,7 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/runtime-corejs2": "^7.0.0",
-    "@elastic/elasticsearch": "^6.2.4",
+    "@elastic/elasticsearch": "^7.0.0",
```

Comment:
So, we won't test our agent against `6.x` versions right? Is it possible to test with both versions?